js/packages/common/src/contexts/meta/meta.tsx

```diff
@@ -38,8 +38,17 @@ export function MetaProvider({
 
   const [state, setState] = useState<MetaState>(getEmptyMetaState());
   const [page, setPage] = useState(0);
-  const [metadataLoaded, setMetadataLoaded] = useState(false);
-  const [lastLength, setLastLength] = useState(0);
+
+  const [
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _metadataLoaded,
+    setMetadataLoaded,
+  ] = useState(false);
+  const [
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _lastLength,
+    setLastLength,
+  ] = useState(0);
```

Comment:
I'm not super sure why this is here if `metadataLoaded` and `lastLength` are never used, but I suspect that the state setters here are being used as a way to force a re-render of the component.